build.cake

```diff
@@ -664,10 +664,17 @@ Task("provision")
     .IsDependentOn("provision-monosdk"); // always provision monosdk last otherwise CI might fail
 
 Task("NuGetPack")
-    .Description("Build and Create Nugets")
-    .IsDependentOn("Restore")
-    .IsDependentOn("BuildForNuget")
-    .IsDependentOn("_NuGetPack");
+    .Description("Build and Create Nugets").Does(()=> {
```

Comment:
This will stop all the builds from running. Maybe we can have a new target that the "packaging" agent does? Then we can have another artifact that we release. I'm worried that something will be skipped. At least keep `BuildForNuget`